gtfs/gtfs_utils/gtfs_utils/core_computations.py

```diff
@@ -187,12 +235,20 @@ def compute_trip_stats(feed: ptg.feed,
 
     # Reset index and compute final stats
     h = h.reset_index()
+
+    # Add rank to duplicate trips and add trip_id_to_date data
+    h['numeric_trip_id'] = h['trip_id'].apply(lambda s: int(s.replace('_', '')))
+    h['trip_id_rank'] = h.groupby(['route_id', 'start_time'])['numeric_trip_id'].rank(method='dense')
+    h = h.merge(trip_to_date, how='inner', on=['route_id', 'start_time', 'trip_id_rank'], validate='one_to_one')
```

Comment:
ooh, I didn't know the validate param, nice. so it will raise an error if not 1:1? We should at least state this in the documentation. 
And also - did this ever happen in your checks? on how many different dates did you try?

Reply:
Yes it raises an error if some value is merged as one_to_many (if the value is not matched there is no error).
It only happened while I was still working on it, I tried a couple of dates (three, I think) and it didn't raise. It really shouldn't happen since we also use rank as a merging column, I put it there more for sanity check.